packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -468,6 +482,9 @@ const DataFilters = ({
                       setIsFiltersMenuOpen(false);
                     }}
                   >
+                    {filterPopoverCurrentFilter?.render &&
+                      filterPopoverCurrentFilter.render(updateFilters)}
```

Comment:
```suggestion
                    {filterPopoverCurrentFilter?.render?.(updateFilters)}
```
Why could render not be a function though? I think all we'd need to do is check that `.render` exists; and the type system forces it to always be a function, so we should be fine with this instead:
```suggestion
                    {filterPopoverCurrentFilter?.render(updateFilters)}
```